packages/file-format/src/lib/file.ts

```diff
@@ -81,11 +81,11 @@ export type TldrawFileParseError =
 
 /** @public */
 export function parseTldrawJsonFile({
-	config,
 	json,
 	instanceId,
+	store,
 }: {
-	config: TldrawEditorConfig
+	store: TLStore
```

Comment:
it looks like we only use the schema from this. would it make more sense to pass the schema directly? Passing in a store and getting back a different store feels a little surprising/weird to me